app/candidates/vendor-marketplace/page.tsx

```
@@ -0,0 +1,193 @@
+  // Fetch candidate's location on component mount
+  useEffect(() => {
+    if (!userId) return;
+    fetch(`/api/candidate?clerkUserId=${userId}`)
+      .then((res) => {
+        if (!res.ok) {
+          throw new Error(`Failed to fetch candidate: ${res.statusText}`);
+        }
+        return res.json();
+      })
+      .then((candidate) => {
+        if (candidate && candidate.city && candidate.state) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)